examples/reagent/project.clj

```diff
@@ -0,0 +1,21 @@
+(defproject todomvc "0.1.0-SNAPSHOT"
+  :dependencies [[org.clojure/clojure "1.6.0"]
```

Comment:
Is the spaces indentation a requirement? All our code should use tabs if that's possible.